packages/rs-drive-nodejs/scripts/build.sh

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env bash
 
```

Comment:
Before we run tests in JS Drive we need to call `yarn build` for rs-drive-nodejs with  CARGO_BUILD_PROFILE=test